_data/people_list.yml

```diff
@@ -464,6 +457,13 @@
   website: https://page.math.tu-berlin.de/~comaneci/
   status: retired
 
+- name: Alexander Demin
+  affiliation: HSE University
+  email: asdemin_2@edu.hse.ru
+  github: sumiya11
+  website: https://sumiya11.github.io/
+  status: retired
```

Comment:
@aaruni96 Not very important, but: why do we shuffle retired contributors to another place in the list? Why not just change the `status` field and be done with it?

Reply:
Exactly @aaruni96 said, for human readability we have individual sections for PI, activate and retired contributors.